index.html

```diff
@@ -0,0 +1,1249 @@
+                </iframe>
+
+                <h2 class="mt-1">JS</h2>
+                <p>
+                    Place the following <span style="color:#89ABE3FF">&lt;script&gt;</span>s near the end of your pages,
+                    right before the closing <span style="color:#89ABE3FF">&lt;/body&gt;</span> tag
+                </p>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%2874%2C118%2C159%2C1%29&t=seti&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cscript%2520src%253D%2522https%253A%252F%252Feasy-ui.netlify.app%252Findex.js%2522%253E%253C%252Fscript%253E"
+                    " class="iframe-container"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </section>
+
+            <!-- Start -->
+
+            <h1 class="mt-2">Components</h1>
+
+            <!-- Avatar -->
+            <section id="avatar" class="mt-1">
+
+                <div class="code-container">
+                    <h2 class="component-title">Avatars</h2>
+                    <a class="avatar">
+                        <img src="https://image.freepik.com/free-vector/businessman-character-avatar-isolated_24877-60111.jpg"
+                            alt="avatar" />
+                    </a>
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2850%2C82%2C109%2C1%29&t=night-owl&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ca%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%253Cimg%2520src%253D%2522https%253A%252F%252Fimage.freepik.com%252Ffree-vector%252Fbusinessman-character-avatar-isolated_24877-%2520%2520%2520%2520%2520%2520%2520%252060111.jpg%2522%2520alt%253D%2522avatar%2522%252F%253E%250A%253C%252Fa%253E"
+                        class="iframe-container"  sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+
+                    <h2>Rounded Avatar</h2>
+                    <a class="avatar rounded">
+                        <img src="https://image.freepik.com/free-vector/businessman-character-avatar-isolated_24877-60111.jpg"
+                            alt="avatar" />
+                    </a>
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2850%2C82%2C109%2C1%29&t=night-owl&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ca%2520class%253D%2522avatar%2520rounded%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522https%253A%252F%252Fimage.freepik.com%252Ffree-vector%252Fbusinessman-character-avatar-isolated_24877-%252060111.jpg%2522%2520alt%253D%2522avatar%2522%252F%253E%250A%253C%252Fa%253E"

```

Comment:
Some of the carbon snippets are not aligned properly.

Reply:
yeah I have tried to fixed that but nothing happened css is not applying on carbon snippets i don't know why